build.gradle

```diff
@@ -0,0 +1,89 @@
+import java.util.regex.Pattern
+
+plugins {
+    id 'java-library'
+}
+
+sourceSets {
+    main {
+        java {
+            srcDirs = [ 'src' ]
+            outputDir = file('classes')
+        }
+    }
```

Comment:
Some resources are missing in runtime. Gradle keeps resources in a separate directory by default. 
Please add this to the main source set:
```
        resources {
            srcDirs = [ 'src' ]
            exclude '*.java', 'package.html'
        }
```